catalog/core/catalog-core-api/src/main/java/org/codice/alliance/catalog/core/api/types/Isr.java

```diff
@@ -27,6 +27,21 @@
     String CLOUD_COVER = "isr.cloud-cover";
 
     /**
+     * Attribute name for accessing the snow cover existence for this Metacard. <br/>
+     */
+    String SNOW_COVER = "isr.snow-cover";
```

Comment:
Could these new constants starting with `S` be moved to the bottom to maintain the (almost) alphabetical order that currently exists in the file?